src/mscorlib/src/System/ArgIterator.cs

```diff
@@ -133,5 +133,54 @@ public override bool Equals(Object o)
             
         private IntPtr ArgPtr;                  // Pointer to remaining args.
         private int    RemainingArgs;           // # of remaining args.
+#else
+        public ArgIterator(RuntimeArgumentHandle arglist)
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator see: https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        [CLSCompliant(false)]
+        public unsafe ArgIterator(RuntimeArgumentHandle arglist, void* ptr)
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        public void End() 
+        { 
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        public override bool Equals(Object o) 
+        {  
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        public override int GetHashCode()
+        { 
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        [System.CLSCompliantAttribute(false)]
+        public System.TypedReference GetNextArg()
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        [System.CLSCompliantAttribute(false)]
+        public System.TypedReference GetNextArg(System.RuntimeTypeHandle rth)
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
+        }
+
+        public unsafe System.RuntimeTypeHandle GetNextArgType()
+        {
+            throw new PlatformNotSupportedException(); //The JIT requires work to enable ArgIterator https://github.com/dotnet/standard/issues/20#issuecomment-272775599.
```

Comment:
Instead of this link, I would have one to an open (or existing) issue in the CoreCLR repo to track this work.